debian/mkchromecast.links

```diff
@@ -1 +0,0 @@
-/usr/share/mkchromecast/mkchromecast.py usr/bin/mkchromecast
```

Comment:
Why was this removed?

Reply:
Well there is no more a `/usr/share/mkchromecast/mkchromecast.py` and because of that a dangling link is created in the resulting deb-file:
```
stefan@mars:~/github$ dpkg-deb -c mkchromecast_0.3.8.1-2_all.deb 
drwxr-xr-x root/root         0 2018-10-21 21:22 ./
drwxr-xr-x root/root         0 2018-10-21 21:22 ./usr/
drwxr-xr-x root/root         0 2018-10-21 21:22 ./usr/bin/
drwxr-xr-x root/root         0 2018-10-21 21:22 ./usr/lib/
[...]
lrwxrwxrwx root/root         0 2018-10-21 21:22 ./usr/bin/mkchromecast -> ../share/mkchromecast/mkchromecast.py
```

I looked into it and found that with commit 3473a77eea7ca989457cbcd13c6c98186dfcf91e mkchromecast was moved directly to bin/mkchromecast, so the file is no longer needed.

I'll add a sentence to my commit.

Reply:
Great :)